boskos/update_prow_config.sh

```diff
@@ -24,20 +24,19 @@ cd "$(git rev-parse --show-toplevel)"
 # Script triggered by prow postsubmit job
 # Update boskos configmap in prow
 
-if [[ -a "${PROW_SERVICE_ACCOUNT:-}" ]] ; then
-	gcloud auth activate-service-account --key-file="${PROW_SERVICE_ACCOUNT}"
+if [[ -a "${PROW_SERVICE_ACCOUNT:-}" ]] ; then # TODO(fejta): delete this in a subsequent PR
```

Comment:
The docs I'm looking at say that `-a` is "and". Should this be `-n` instead?

I know this is the same as it was before, but I don't know why it was that way, either.

Reply:
`-a` means file exists: http://tldp.org/LDP/Bash-Beginners-Guide/html/sect_07_01.html